client/scss/components/_listing.scss

```diff
@@ -487,6 +487,10 @@ table.listing {
       font-size: 0.8em;
     }
 
+    .locked-indicator-owner {
```

Comment:
I recommend we go with the current `--color-primary` (light teal), and as per @thibaudcolas 's comment we will also need a visual indicator aside from the colour.

We want something subtle, so a small circle border will be suitable, however we do not want this to change the height of the icon container so let's use a simple box shadow as a border.

See below for proposed change.

```scss

    .locked-indicator {
      font-size: 0.8em;
      border-radius: 50%;
    }

    .locked-indicator-owner {
      color: var(--color-primary);
      box-shadow: 0 0 0 1px var(--color-primary);
    }
```

<img width="721" alt="Screen Shot 2022-05-06 at 7 33 12 am" src="https://user-images.githubusercontent.com/1396140/167029157-0fcfa08f-716a-49dc-8897-90192d174fc5.png">